db/migrate/20210111145453_create_openstax_accounts_accounts.openstax_accounts.rb

```diff
@@ -0,0 +1,23 @@
+# This migration comes from openstax_accounts (originally 0)
+class CreateOpenStaxAccountsAccounts < ActiveRecord::Migration[4.2]
```

Comment:
I see some of these say `ActiveRecord::Migration[6.0]` and some say `ActiveRecord::Migration[4.2]` - what's the difference? Can this cause issues?

Reply:
These are migrations from the gems that have database tables. I'm not sure what the migration number is referring to.

Reply:
The number is just what Rails version the migration was created in, it affects things such as the default column type used for foreign keys... maybe other things.